front/pages/api/v1/me.ts

```diff
@@ -31,6 +31,4 @@ async function handler(
   }
 }
 
-export default withAuth0TokenAuthentication(handler, {
-  requiredScopes: { GET: "read:user_profile" },
```

Comment:
do we need to remove the requiredScopes here ? for workos tokens, we should not check the scope anyway